dnscrypt-proxy/proxy.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/jedisct1/dlog"
 	clocksmith "github.com/jedisct1/go-clocksmith"
 	stamps "github.com/jedisct1/go-dnsstamps"
+	"github.com/miekg/dns"
```

Comment:
I know nothing about Go, but am wondering if pulling in a fairly large library for one rcode is worth it or has any negative side effects (increase binary size).

Reply:
It's already used for many more things in other files :)